lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.h

```diff
@@ -23,6 +26,8 @@ extern "C" {
 namespace magma {
 namespace lte {
 
+extern task_zmq_ctx_t task_zmq_ctx_main_spgw;
```

Comment:
From what I understand extern declaration should be done in header files not source.